src/Clock.elm

```diff
@@ -0,0 +1,103 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Html exposing (..)

```

Comment:
Avoid wildcard imports.
It makes a huge difference when you can easily see where each symbol comes from.